package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "app.js",
   "scripts": {
-    "test": "mocha",
+    "test": "mocha --exit",
```

Comment:
This is not the right way to fix this; your test is not exiting because there are still things running. Most likely it is express and mongoose.

I'd expect adding the proper cleanup hooks will allow your tests to exit: https://mochajs.org/#hooks